cmd/ferretdb/main.go

```diff
@@ -204,9 +214,86 @@ var (
 
 func main() {
 	setCLIPlugins()
-	kong.Parse(&cli, kongOptions...)
+	ctx := kong.Parse(&cli, kongOptions...)
+
+	switch ctx.Command() {
+	case "run":
+		run()
+	case "ping":
+		ping()
+	default:
+		panic("unknown sub-command")
+	}
+}
+
+// ping creates connection to FerretDB instance specified by the flags, and runs `ping` command against it.
+func ping() {
+	logger := setupLogger(setupState(), cli.Log.Format)
+	l := logger.Sugar()
+
+	checkFlags(logger)
+
+	if cli.Setup.Database == "" {
+		return
+	}
+
+	if cli.Listen.Addr != "" {
+		host, port, err := net.SplitHostPort(cli.Listen.Addr)
+		if err != nil {
+			l.Fatal(err)
+		}
+
+		if host == "" {
+			host = "127.0.0.1"
+		}
+
+		u := &url.URL{
+			Scheme: "mongodb",
+			Host:   fmt.Sprintf("%s:%s", host, port),
+			Path:   cli.Setup.Database,
+			User:   url.UserPassword(cli.Setup.Username, cli.Setup.Password),
+		}
+
+		ctx, cancel := context.WithTimeout(context.Background(), cli.Setup.Timeout)
+		defer cancel()
 
-	run()
+		client, err := mongo.Connect(ctx, options.Client().ApplyURI(u.String()))
+		if err != nil {
+			l.Fatal(err)
+		}
+
+		defer func() {
+			if err = client.Disconnect(ctx); err != nil {
+				l.Fatal(err)
+			}
+		}()
+
+		if err = client.Ping(ctx, nil); err != nil {
+			l.Fatal(err)
+		}
+	}
+
+	if cli.Listen.Unix != "" {
+		ctx, cancel := context.WithTimeout(context.Background(), cli.Setup.Timeout)
+		defer cancel()
+
+		u := strings.ReplaceAll(cli.Listen.Unix, "/", "%2F")
+
+		client, err := mongo.Connect(ctx, options.Client().ApplyURI("mongodb://"+u))
+		if err != nil {
+			l.Fatal(err)
+		}
+
+		defer func() {
+			if err = client.Disconnect(ctx); err != nil {
+				l.Fatal(err)
+			}
+		}()
+
+		if err = client.Ping(ctx, nil); err != nil {
+			l.Fatal(err)
+		}
+	}
```

Comment:
Okay, if neither `cli.Listen.Addr` or `cli.Listen.Unix` are specified, shouldn't we return error? I guess there is TLS case as well.